src/resources/train.py

```diff
@@ -0,0 +1,55 @@
+import pandas as pd
+import torch
+from datasets import Dataset
+from transformers import BertForSequenceClassification, BertTokenizer, Trainer, TrainingArguments
```

Comment:
add to the pyproject.toml in `[tool.poetry.dependencies]` 
```
torch = "^actual_version"
transformers = "^actual_version"
```
and run `poetry lock` in termianl, after that push this commmit